### DISCUSSION TITLE
Daily 08/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#559] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve tratando de realiazar el backend en django para la página de perfil. No pude hacer que funcione, así que hoy continuaré con esa tarea. Por la tarde tengo clase de programación.

**Comment by rpgrca** [comment]
Ayer terminé todo lo de ventas, también el listado de ventas del administrador, hoy voy a hacer el listado de compras del usuario, y voy a empezar con el ABM de envíos que se necesita para poder probar la vuelta de las ventas (si no hay que agregarlas a mano en la base de datos). Y seguramente siga con la parte de ofertas.

**Comment by BraianTroncoso** [comment]
Ayer estuve con el is_staff y me costó, también estuve con meet de django, y trabajo aparte.
